_docs/releasing_version.md

```diff
@@ -31,6 +31,15 @@ And in case you installed the gems before:
 bundle install --path .vendor/ --without system_tests development; bundle update; bundle clean
 ```
 
+If the module contains a Puppet Strings generated documentation, please
+ensure the file is update to date. A good indicator for a Puppet Strings
+documentation is the existence of a REFERENCE.md file. You can automatically
+generate the documentation by running the following rake task:
+
+```bash
+bundle exec rake strings:generate:reference
```

Comment:
We also have a task `reference` which at least in puppet-extlib is used to set the patterns, though it's not needed right now. Perhaps we should just remove that and enhance the task if needed.